tools/yulPhaser/main.cpp

```diff
@@ -45,46 +45,12 @@ int main(int argc, char** argv)
 
 		std::cerr << std::endl;
 		std::cerr << "ERROR: " << exception.what() << std::endl;
-		return 1;
-	}
-	catch (solidity::util::Exception const& exception)
-	{
-		// Something's seriously wrong. Probably a bug in the program or a missing handler (which
-		// is really also a bug). The exception should have been handled gracefully by this point
-		// if it's something that can happen in normal usage. E.g. an error in the input or a
-		// failure of some part of the system that's outside of control of the application (disk,
-		// network, etc.). The bug should be reported and investigated so our job here is just to
-		// provide as much useful information about it as possible.
-
-		std::cerr << std::endl;
-		std::cerr << "UNCAUGHT EXCEPTION!" << std::endl;
-
-		// We can print some useful diagnostic info for this particular exception type.
-		std::cerr << "Location: " << exception.lineInfo() << std::endl;
-
-		char const* const* function = boost::get_error_info<boost::throw_function>(exception);
-		if (function != nullptr)
-			std::cerr << "Function: " << *function << std::endl;
-
-		// Let it crash. The terminate() will print some more stuff useful for debugging like
-		// what() and the actual exception type.
-		throw;
-	}
-	catch (std::exception const&)
-	{
-		// Again, probably a bug but this time it's just plain std::exception so there's no point
-		// in doing anything special. terminate() will do an adequate job.
-		std::cerr << std::endl;
-		std::cerr << "UNCAUGHT EXCEPTION!" << std::endl;
-		throw;
+		return 2;
 	}
 	catch (...)
 	{
-		// Some people don't believe these exist.
-		// I have no idea what this is and it's flying towards me so technically speaking it's an
-		// unidentified flying object.
-		std::cerr << std::endl;
-		std::cerr << "UFO SPOTTED!" << std::endl;
-		throw;
+		std::cerr << "Uncaught exception:" << std::endl;
```

Comment:
Any particular reason why we're not using `using namespace std` here, and more importantly, should we open a separate issue to address, or do it in this PR? Or do we want to just leave it as is? @cameel, I understand you've done most of the work on the phaser, so I guess it's up to you.

Reply:
Also, if we have two consecutive stream writes, it would make more sense to append a newline, i.e. `cerr << "bla bla" << "\n"` and only `endl` in the last write, as there's no need to flush the stream every single time. We seem to follow this pattern everywhere though, so this PR is probably not the best place to fix it.